src/Providers/Mistral/Client.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EchoLabs\Prism\Providers\Mistral;
+
+use Illuminate\Http\Client\PendingRequest;
+use Illuminate\Http\Client\Response;
+use Illuminate\Support\Facades\Http;
+
+class Client
+{
+    protected PendingRequest $client;
+
+    public function __construct(
+        public readonly string $url,
+        public readonly ?string $apiKey,
```

Comment:
We should probably update things so that the API key is required.

Reply:
Pushed the changes. [9802f42](https://github.com/echolabsdev/prism/pull/13/commits/9802f423c1b91da29b93098baab58e8e505dd8b5)